Apps/Playground/Scripts/experience.js

```diff
@@ -177,6 +178,73 @@ CreateBoxAsync().then(function () {
                     }, 5000);
                 }
 
+                // Showing visualization for ARKit LiDAR mesh data
+                if(meshDetection) {
+                    var mat = new BABYLON.StandardMaterial("mat", scene);
+                    mat.wireframe = true;
+                    mat.diffuseColor = BABYLON.Color3.Blue();
+                    const xrMeshes = xr.baseExperience.featuresManager.enableFeature(
+                    BABYLON.WebXRFeatureName.MESH_DETECTION,
+                    "latest",
+                    {convertCoordinateSystems: true});
+                    console.log("Enabled mesh detection.");
+                    const meshes = [];
+                    const meshID = [];
```

Comment:
Using these two arrays seems a little odd.  Maybe consider just using a map of mesh IDs to meshes.

eg.
const map = new Map();

// Add mesh to map (onMeshAddedObservable)
meshMap.set(meshId, customMesh);

// get mesh from map (onMeshUpdatedObservable)
if (meshMap.has(mesh.id)) {
   const mesh = meshMap[mesh.id];
}

// Delete mesh from map onMeshRemovedObservable
meshMap[mesh.id].dispose();
meshMap.delete(mesh.id);

Reply:
Sounds good, will make the change